tests/Basics/RandomTest.cpp

```diff
@@ -22,14 +22,53 @@
 /// @author Copyright 2007-2012, triAGENS GmbH, Cologne, Germany
 ////////////////////////////////////////////////////////////////////////////////
 
-#include "Basics/Common.h"
-
 #include "gtest/gtest.h"
 
 #include "Random/RandomGenerator.h"
+#include <stdint.h>
```

Comment:
This is a C-style include. Please use the C++ style includes.
```suggestion
#include <cstdint>
```